client/packages/coldchain/src/Equipment/api/hooks/document/useAsset.ts

```diff
@@ -1,5 +1,4 @@
-import { useMutation, useParams } from '@openmsupply-client/common';
-import { useQuery } from 'react-query';
```

Comment:
offroad - noticed that some files were importing directly from react-query which they should not!